test/promql/tests/data_plane/availability_metric_konflux_test.yaml

```diff
@@ -0,0 +1,36 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.availability_metric_konflux_alerts.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: 'konflux_up{service="grafana", check="prometheus-appstudio-ds", source_cluster="stone-stg-rh01"}'
+        values: '1x59 _x61'
```

Comment:
Please add more input series or additional tests to exercise the scenario being discussed in the other [thread](https://github.com/redhat-appstudio/o11y/pull/262#discussion_r1585476819).